### ISSUE TITLE
Append more values to sharedStrings of workbook

### ISSUE THREAD

**Issue by NayOoLwin5** [optilude/xlsx-template#164]
My workbook is already loaded with values and placeholders. The total of sharedStrings is now 530. I would like to add(append) more values to the sharedStrings along with a loop.

**Comment by kant2002** [comment]
This is library for report generation, why do you want to put values into `sharedStrings` using it?

**Comment by kant2002** [comment]
Aah, now it make sense, so you essentially want stream substitution process. And be able append more substitution values over and over again until you done. Is this correct?

# Example
```js
var batch1 = [
    {name: "John Smith", age: 20},
    {name: "Bob Johnson", age: 22}
];

// Perform substitution
template.substitute(sheetNumber, values);

var batch2 = [
    {name: "Andrii Hopak", age: 33},
    {name: "Marat Beysabekov", age: 52}
];

// Feed additional data to substitution process
template.continueSubstitute(sheetNumber, values);
```

**Comment by kant2002** [comment]
There no such functionality in the library at the moment. You have to collect all data into one object, and if you are done, then perform substitution.

**Comment by NayOoLwin5** [comment]
Understood @kant2002. In addition, does this library support merging multiple excel files into one? If it does, it solves my issue.

**Comment by kant2002** [comment]
No, merging of XLSX files should happens via other means. This library is for simple template substitution.
